tests/acceptance/test_node_operators_registry.py

```diff
@@ -140,25 +121,12 @@ def test_nor_state(contract):
         assert node_operator_summary["stuckValidatorsCount"] == 0
         assert node_operator_summary["refundedValidatorsCount"] == 0
         assert node_operator_summary["stuckPenaltyEndTimestamp"] == 0
-        # counts could inc but not dec
-        if id == 2:
-            assert node_operator_summary["totalExitedValidators"] == 252
-        elif id == 4:
-            assert node_operator_summary["totalExitedValidators"] == 174
-        elif id == 5:
-            assert node_operator_summary["totalExitedValidators"] == 36
-        elif id == 12:
-            assert node_operator_summary["totalExitedValidators"] == 2300
-        elif id == 21:
-            assert node_operator_summary["totalExitedValidators"] == 125
-        elif id == 22:
-            assert node_operator_summary["totalExitedValidators"] == 11
-        else:
-            assert (
-                node_operator_summary["totalExitedValidators"] == 0
-            ), f"totalExitedValidators is positive for node {id}"
 
-        assert node_operator_summary["totalDepositedValidators"] > 0
+        # Invariant check
+        # https://github.com/lidofinance/lido-dao/blob/cadffa46a2b8ed6cfa1127fca2468bae1a82d6bf/contracts/0.4.24/nos/NodeOperatorsRegistry.sol#L168
+        assert node_operator_summary["totalExitedValidators"] >= 0
+        assert node_operator_summary["totalExitedValidators"] <= node_operator_summary["totalDepositedValidators"]
+
```

Comment:
Let's also add a check that values returned by the `getNodeOperator()` and `getNodeOperatorSummary()` match. The values we can check must be: `exited`, `deposited`, and `depositable` validators. If I'm not mistaken, the last one might be asserted as `vetted - deposited == depositable`.

Reply:
Agree, good idea!